engine/apps/slack/tests/test_scenario_steps/test_alert_group_actions.py

```diff
@@ -271,6 +271,7 @@ def test_get_alert_group_from_slack_message_in_db_no_alert_group(
         step.get_alert_group(slack_team_identity, payload)
 
 
+@pytest.mark.xdist_group(name="group1")
```

Comment:
This is to make sure all these tests reusing the same organization ID, alert group ID, etc. are run in the same worker, sequentially. Otherwise, a parallel run can make them [fail](https://github.com/grafana/oncall/actions/runs/8554723710/job/23441071757?pr=4153#step:5:648). See `loadgroup` [param](https://pytest-xdist.readthedocs.io/en/latest/distribution.html) below.